dependencies.yaml

```diff
@@ -406,6 +407,26 @@ dependencies:
               - *libcusolver114
               - *libcusparse_dev114
               - *libcusparse114
+      - output_types: pyproject
```

Comment:
I am guessing that you wrote `pyproject` and not `[requirements, pyproject]` because the `requirements` output would affect devcontainers, and we don't want that? Is there a better way to solve this?

Reply:
I honestly didn't really think about devcontainers. Not sure what we want to do there. cc @trxcllnt

Reply:
I think the simplest options are:
- use `pyproject` only (and not `requirements`) because devcontainers would pick up the `requirements` entries and we don't want those in devcontainers
- use `[requirements, pyproject]` and have devcontainers post-process the requirements list to exclude CUDA wheels

Reply:
The first option makes the most sense to me, and it's already implemented here :)